api/app.js

```diff
@@ -29,7 +29,6 @@ const dsRouter = require('./dsService/dsRouter');
 const recipientRouter = require('./recipient/recipientRouter');
 const householdRouter = require('./household/householdRouter');
 const metricsRouter = require('./metrics/metricsRouter');
-const veteransRouter = require('./veteran/veteranRouter');
```

Comment:
Hopefully everything will work now!

Reply:
Yep that was exactly it. npm start now works locally at least. Didn't notice before because the unit tests didn't run it. Something to look out for in the future